shared/entities/project-key-costs.view.ts

```diff
@@ -0,0 +1,117 @@
+import { ValueTransformer, ViewColumn, ViewEntity } from "typeorm";
+
+export const decimalTransformer: ValueTransformer = {
```

Comment:
This utility already exists in:
https://github.com/Vizzuality/tnc-blue-carbon-cost-tool/blob/dev/shared/entities/base-data.view.ts#L4

You can either import it from there, or move the existing one to a more ergonomic place, up to you, but let's not duplicate it